python_ta/typecheck/base.py

```diff
@@ -212,8 +212,9 @@ def lookup_concrete(self, tvar):
     ### HELPER METHODS
     def types_in_callable(self, callable_function):
         """Return a tuple of types corresponding to the Callable function's arguments and return value, respectively."""
-        types_lst = [arg_type_lst.append(self.lookup_concrete(argument)) for argument in callable_function.__args__]
-        return types_lst[:-1], types_lst[-1]
+        arg_type_lst = []
+        [arg_type_lst.append(self.lookup_concrete(argument)) for argument in callable_function.__args__]
```

Comment:
Sorry I missed this last time. You should not use `append` here. Just do `types_lst = [self.lookup_concrete(argument) for ...]`